crates/node/src/lib.rs

```diff
@@ -198,24 +206,27 @@ impl Node {
             .await?;
 
         let nodes_config = metadata.nodes_config();
+
         // Find my node in nodes configuration.
-        let Some(current_config) = nodes_config.find_node_by_name(&self.options.node_name) else {
+        let Some(my_node_config) = nodes_config.find_node_by_name(&self.options.node_name) else {
             // Node is not in configuration. This is currently not supported. We only support
             // static configuration.
             bail!("Node is not in configuration. This is currently not supported.");
```

Comment:
This probably can be removed now, we can add the node to the config if it has a new name

Reply:
Will remove it.